docs/program/sdks/_index.md

```diff
@@ -227,9 +206,9 @@ The C++ SDK is currently in alpha.
 using namespace viam::sdk;
 
 int main() {
-  std::string host("ADDRESS FROM THE VIAM APP");
+  std::string host("rover-main-main.yriemen2i2.viam.cloud");
   DialOptions dial_opts;
-  Credentials credentials("SECRET FROM THE VIAM APP");
+  Credentials credentials("d7fxn4zc92cjwmlhd7zh545la0xmpc71nnn9gmtyb28g0nhd");
```

Comment:
Since we're instructing users to show the secret now, this will appear populated. Elsewhere as well.

Reply:
If you want to show something show a credential that is recognizably fake like `a1bcd2ef34ghijklmn5op678qr9stuvw01xyz2abcde12f3ghi` but I think it's easier for readers to see what they need to change if we keep the instructions.

You can also move the instruction to get the secret and address to the next step after the code and add some instruction to store the credentials safely.